include/packager/live_packager.h

```diff
@@ -124,10 +138,10 @@ class LivePackager {
   /// @return OK on success, an appropriate error code on failure.
   Status Package(const Segment& init_segment,
                  const Segment& media_segment,
-                 FullSegmentBuffer& output);
+                 SegmentBuffer& output);
 
   Status PackageTimedText(const Segment& media_segment,
-                          FullSegmentBuffer& output);
+                          SegmentBuffer& output);
```

Comment:
I am not sure how this change could work for timed-text packaging.  With this new simplified interface we lose information on the init segment.

Reply:
We're not using this data (init segment saving) _anywhere_ in SIM at the moment. There are no tests that use this functionality. When we need it, I will add a separate API call, just like we do for A/V segments, to package timed text init segment

Reply:
Here's a piece of code from SIM:
```
tt.initMeta = metadata
	if metadata.Output.Type == model.ContainerTypeTS {
		return nil, nil
	}

	data, err := segment.Encode()
	if err != nil {
		return nil, err
	}
```
We just encode the given boxes from mp4 reader. No need to package it with shaka